excretor/src/tummy.rs

```diff
@@ -72,7 +72,7 @@ impl Tummy {
         )
         .fetch_one(&self.tummy_conn_pool)
         .await?;
-        Ok(models::Channel::from_db_channel(&channel))
+        Ok(models::Channel::from(&channel))
```

Comment:
you can directly do `&channel.into()` hehe.

Reply:
```
the trait bound `DBChannel: Into<_>` is not satisfied
required for `DBChannel` to implement `Into<Channel>`
```
hmmmmm

Reply:
I am most certainly wrong in its interpretation, but check [4e2a871](https://github.com/kossiitkgp/top-secret-stuff/pull/45/commits/4e2a871cb0302351fff3e092e30779877e16974b) once, and suggest

Reply:
![image](https://github.com/kossiitkgp/top-secret-stuff/assets/34770591/038fea76-994a-4f06-98a3-c128d7f38e24)
It should work?

Reply:
maybe you need to implement it for `DBChannel` rather than `&DBChannel`?

Reply:
Or maybe it is because it converts a `&DBChannel` into a `Channel` so for the reverse you need to run `.into()` on a `Channel` not a `&Channel`.

Reply:
`Ok((&channel).into())` this works.

Reply:
You can just put the `&` operator before the `query_as!` I think.